packages/core/src/components/alert/alert.spec.ts

```diff
@@ -257,4 +260,32 @@ describe('AtomAlert', () => {
 
     expect(spy).toHaveBeenCalled()
   })
+
+  it('should sanitize the messageTitle and messageText using DOMPurify', async () => {
+    const sanitizedText = '<i>Sanitized Text</i>'
+    const sanitizedTitle = '<strong>Sanitized Title</strong>'
+
+    ;(DOMPurify.sanitize as jest.Mock).mockImplementation((input: string) => {
```

Comment:
Guess it doesn't help too much, man. Once there is no logic on our side, this test would help more if we don't mock the library.

Reply:
It is not feasible to test this behavior without using mocks, as DOMPurify performs sanitization at runtime.

Like you said, mocking in this context is not appropriate, as it would essentially mean testing the internal behavior of an external library. Therefore, I decided to remove the tests: https://github.com/juntossomosmais/atomium/pull/749/commits/ae8602de638e6f6bd953d9d265f841b0b0599bb0

I will continue to rely on the robustness of the library itself, especially considering that its test suite is extensive and trustworthy: https://github.com/cure53/DOMPurify/blob/main/test/test-suite.js

Reply:
It makes sense!